src/index.tsx

```diff
@@ -1,5 +1,10 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <BrowserRouter>
```

Comment:
use `HashRouter` instead, cause now if I reload page on `gh-pages`, there is the 404 error